Methylseq/methylseq.sh

```diff
@@ -228,43 +308,41 @@ if [ ! -f "$temp_path/$unmethyl_control/$hydroxymethyl_control/genome_alignment/
         do
             chr="${chromosome}${i}"
             echo "$chr"
-            samtools view -b "${trimmed_R1_file_name}.fastq.gz_unmapped_reads_1.fq.gz_unmapped_reads_1_bismark_bt2_pe.bam.sorted.bam" $chr > "split_bams/${trimmed_R1_file_name}.fastq.gz_unmapped_reads_1.fq.gz_unmapped_reads_1_bismark_bt2_pe.bam.${chr}.${sample_name}.bam"  
+            samtools view -b "${trimmed_R1_file_name}.fastq.gz_unmapped_reads_1.fq.gz_unmapped_reads_1.fq.gz_unmapped_reads_1_bismark_bt2_pe.deduplicated.bam.sorted.bam" $chr > "split_bams/${trimmed_R1_file_name}.fastq.gz_unmapped_reads_1.fq.gz_unmapped_reads_1.fq.gz_unmapped_reads_1_bismark_bt2_pe.deduplicated.bam.${chr}.${sample_name}.bam"  
         done
-    rsync -vur $temp_path/ $seq_path
+    rsync -vur --exclude "main_genome" --exclude "unmethyl_genome" --exclude "hydroxymethyl_genome" --exclude "methyl_genome" $temp_path/ $seq_path
     echo "bam file has been split up by chromosome"
 else
     echo "bam file was already split up by chromosome"
 fi
 
-#####################previous insert_size_analysis.sh###########################
+#####################previously insert_size_analysis.sh###########################
 ################################################################################
-if [ ! -f "$temp_path/$unmethyl_control/$hydroxymethyl_control/genome_alignment/${trimmed_R1_file_name}.fastq.gz_unmapped_reads_1.fq.gz_unmapped_reads_1_bismark_bt2_pe.bam_picard_insert_size_plot.pdf" ]; then 
-   cd "$temp_path/$unmethyl_control/$hydroxymethyl_control/genome_alignment"
-        echo "$temp_path/$unmethyl_control/$hydroxymethyl_control/genome_alignment/${trimmed_R1_file_name}.fastq.gz_unmapped_reads_1.fq.gz_unmapped_reads_1_bismark_bt2_pe.bam_picard_insert_size_plot.pdf"
+if [ ! -f "$temp_path/$unmethyl_control/$hydroxymethyl_control/$methyl_control/genome_alignment/${trimmed_R1_file_name}.fastq.gz_unmapped_reads_1.fq.gz_unmapped_reads_1.fq.gz_unmapped_reads_1_bismark_bt2_pe.bam_picard_insert_size_plot.pdf" ]; then 
```

Comment:
the insert size analysis we also want to use sorted and indexed deduplicated files.